src/com/wolox/parser/ConfigParser.groovy

```diff
@@ -59,11 +59,14 @@ class ConfigParser {
         def services = [];
 
         steps.each {
-            def instance = getServiceClass(it.capitalize())?.newInstance()
-            services.add(instance)
+            def service = it.tokenize(':')
+            def version = service.size() == 2 ? service.get(1) : 'latest'
+            def instance = getServiceClass(service.get(0).capitalize())?.newInstance()
+
+            services.add([service: instance, version: version])
```

Comment:
Another alternative is move the version to the service class https://github.com/Wolox/wolox-ci/pull/5/files#diff-0a3510f7b6882a0558990026952287fdR74